packages/react-dialog/src/components/Dialog/useDialogStyles.ts

```diff
@@ -1,8 +1,14 @@
+import { SlotClassNames } from '@fluentui/react-utilities/src';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { DialogState } from './Dialog.types';
+import type { DialogSlots, DialogState } from './Dialog.types';
 
+/**
+ * @deprecated
+ */
 export const dialogClassName = 'fui-Dialog';
```

Comment:
For `Dialog` you can actually remove this class name instead of deprecating it since the component is not being exported from `@fluentui/react-components` yet:

```suggestion
```